tests/Controller/ExportControllerTest.php

```diff
@@ -769,4 +774,141 @@ public function testExportHeaders(array $expectedHeaders): void
 
         $this->assertEquals($expectedHeaders, $headers);
     }
+
+    public function exceptionMessageProvider()
+    {
+        return [
+            'EmptyValueNotAllowedException is logged' => [
+                'Product with id "%s" could not be exported. It appears to have empty values assigned to it. ' .
+                'If you see this message in your logs, please report this as a bug.'
+            ],
+            'General Exception is logged' => [
+                'Error while exporting the product with id "%s". If you see this message in your logs, ' .
+                'please report this as a bug. Error message: Test Exception'
+            ]
+        ];
+    }
+
+    /**
+     * @dataProvider exceptionMessageProvider
+     */
+    public function testExceptionIsNotLogged(string $exceptionMessage): void
```

Comment:
What is this test? I tried reading it a couple of times but it does not seem to make sense. If an exception is thrown, we want to catch it and log it. But here you expect nothing but be logged, so why would you even care about the message, if it is never called anyway? Seems odd. Also the data providers do not change how the test runs, therefore they will both return the same result.